app/callbacks/data_callbacks.py

```diff
@@ -142,67 +136,153 @@ def login_callback(n_clicks, username, password, user_headers, lang):
                     empty_style_unchanged,
                     hide_element_style,
                 )
-
     raise PreventUpdate
 
 
 @app.callback(
     [
-        Output("store_api_alerts_data", "data"),
+        Output("store_wildfires_data", "data"),
+        Output("store_detections_data", "data"),
+        Output("media_url", "data"),
+        Output("trigger_no_wildfires", "data"),
+        Output("previous_time_event", "data"),
     ],
-    [Input("main_api_fetch_interval", "n_intervals"), Input("user_credentials", "data")],
+    [Input("main_api_fetch_interval", "n_intervals")],
     [
-        State("store_api_alerts_data", "data"),
-        State("user_headers", "data"),
+        State("client_token", "data"),
+        State("media_url", "data"),
+        State("store_wildfires_data", "data"),
+        State("previous_time_event", "data"),
     ],
     prevent_initial_call=True,
 )
-def api_watcher(n_intervals, user_credentials, local_alerts, user_headers):
+def data_transform(n_intervals, client_token, media_url, store_wildfires_data, previous_time_event):
```

Comment:
this is way more complexe than the current code, is fetch_unlabeled_detections different from get_unacknowledged_events in old-production branch ?

Reply:
no these functions are the same but before we are gathering events and showing events in the interface, now we are gathering detections and showing wildfire so we need to construct the wildfire object online